deeplinkdispatch-processor/src/main/java/com/airbnb/deeplinkdispatch/DeepLinkProcessor.java

```diff
@@ -534,4 +541,57 @@ private void generateDeepLinkDelegate(String packageName, List<TypeElement> load
         .build()
         .writeTo(filer);
   }
+
+  private void generateDeepLinkDoc(List<DeepLinkAnnotatedElement> elements) {
```

Comment:
This file is getting kind of large. Would be nice to encapsulate all the doc generation in a separate class. You can pass in the file name in the constructor and call `docGenerator.generateDoc(List<DeepLinkAnnotatedElement>)`. This also has the advantage that you can unit test the DocGenerator on its own without having to construct a sample project with all of the annotations.